#!/bin/sh /usr/share/dpatch/dpatch-run
## 337-magic-add-tokyocabinet.dpatch by Benoit Sibaud <bsibaud@april.org>
##
## DP: New magic for TokyoCabinet database (Closes: #481768).

@DPATCH@

diff -Naurp file-4.24.orig/magic/Magdir/database file-4.24/magic/Magdir/database
--- file-4.24.orig/magic/Magdir/database	2008-05-21 09:07:49.000000000 +0000
+++ file-4.24/magic/Magdir/database	2008-05-21 09:15:26.000000000 +0000
@@ -247,3 +247,9 @@
 # From:	Benoit Sibaud <bsibaud@april.org>
 0	string	\\[depot\\]\n\f		Quick Database Manager, little endian
 0	string	\\[DEPOT\\]\n\f		Quick Database Manager, big endian
+
+# Type:	TokyoCabinet database
+# URL:	http://tokyocabinet.sourceforge.net/
+# From:	Benoit Sibaud <bsibaud@april.org>
+0	string	ToKyO\ CaBiNeT\n	TokyoCabinet database
+>14	string	x			(version %s)
